johnwayodi/store-manager
# #161166440 Add Admin view for All Store Attendants

#### What does this PR do?
Adds view where Admin can see all Store Attendants in the store manager application
#### Description of Task to be completed?
Ensure the individual Store Attendant items are clickable and open a Sales attendant details view
Ensure the _**create new attendant**_ option opens a view where admin can create new store attendant
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/store-manager.git`
2. Ensure you are in the directory where you cloned this repository, **_store-manager_** by default
3. `git checkout ft-all-attendants-admin-161166440`
4. open all-attendants.html on browser
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#161166440
#### Screenshot
![all-attendants](https://user-images.githubusercontent.com/10683903/46838946-374edf00-cdc4-11e8-8fd6-aa43f21948ef.png)

#### Questions:
Any feedback on what I can improve or change? Drop a comment or suggestion below
